en/docs/learn/design-api/publish-api/publish-an-api-with-artifact-synchronization.md

```diff
@@ -0,0 +1,158 @@
+#Publish an API with Artifact Synchronization
```

Comment:
The intended functionality is publishing APIs with artifact synchronization. Thus this will be the apt heading.  And according to the offline discussion with  @pubudu538 we are placing this in install-and-setup/setup/distributed-deployment. we can change the position of this doc later

Reply:
Where do we have docs mentioning about gateway labels, what they are, how to create them and how they are in effect? So far we only have it in microgateway.

Reply:
Please add thse configs to the config catalog.